changelog.d/pr-10728

```diff
@@ -0,0 +1,11 @@
+synopsis: Fix `file+noindex` URI usage on Windows
+packages: cabal-install
+prs: #10728
+issues: #10703
+significance:
+
+description: {
+
+- Windows users can now specify file+noindex paths using the format `file+noindex:////./C:/path/to/repo`.

```

Comment:
It’d be good to add a summary of the discussion on the ticket, perhaps. Many people will see `//./` for the first time in their life, I bet. At least, a reference introducing this thing could be helpful.

Reply:
I added a small explanation in the changelog fragment.

Reply:
Terrific, thank you! One small suggestion below.